docs/admin/shipping-bytecode.md

```diff
@@ -57,8 +57,21 @@ To use the provided templated Containerfile simply run a `podman build` command
 like the following:
 
 ```bash
-podman build  --build-arg PROGRAM_NAME=xdp_counter --build-arg SECTION_NAME=pass --build-arg PROGRAM_TYPE=xdp --build-arg BYTECODE_FILENAME=pass.bpf.o --build-arg KERNEL_COMPILE_VER=$(uname -r) -f packaging/container-deployment/Containerfile.bytecode /home/<USER>/bytecode -t quay.io/<USER>/xdp_pass:latest
+podman build \
+ --build-arg PROGRAM_NAME=xdp_counter \
+ --build-arg SECTION_NAME=pass \
+ --build-arg PROGRAM_TYPE=xdp \
+ --build-arg BYTECODE_FILENAME=pass.bpf.o \
+ --build-arg KERNEL_COMPILE_VER=$(uname -r) \
+ -f packaging/container-deployment/Containerfile.bytecode \
+ /home/<USER>/bytecode -t quay.io/<USER>/xdp_pass:latest
 ```
+Where `/home/<USER>/bytecode` is the directory the bytecode object file is located.
+
+
+**NOTE:** By default, `podman` will use the OCI media type as described above.
```

Comment:
I am using Docker version 20.10.18 and I couldn't get it to work. I had podman on my laptop and it worked.

Reply:
```
$ skopeo inspect --raw docker://quay.io/billy99/gocounter-bytecode:latest
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
   "config": {
      "mediaType": "application/vnd.docker.container.image.v1+json",
      "size": 2998,
      "digest": "sha256:e9a5646e514160b53af156f8c1c623c6469db25ecb5ba78083f4a0a1368ef3d3"
   },
   "layers": [
      {
         "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
         "size": 2642,
         "digest": "sha256:e3712661920c77635cdab75372e4f331c7cf4b7677355ff2992dfd055e37aca0"
      }
   ]
}
```

Reply:
Huh that's weird! it's using it's own Media type there... I was running docker 1.19 but let me try again

Reply:
I'm seeing the same thing...

https://docs.docker.com/registry/spec/manifest-v2-2/

Reply:
We can change bpfd to account for this since it's still just a gzipped Tar, but It's pretty annoying they're not using the media types from https://github.com/opencontainers/image-spec/blob/main/media-types.md

Reply:
Lol https://github.com/containers/skopeo/issues/1692

Reply:
It's all good though I guess https://github.com/opencontainers/image-spec/blob/main/media-types.md#applicationvndociimagelayerv1targzip 

As it's explicitly labeled as compatible in the spec

Reply:
I'll fix that now, but I think you can remove that note still :)

Reply:
Fixed on next PR.